Wikipedia/Code/WebViewController.m

```diff
@@ -687,8 +687,11 @@ - (void)updateReferencesHeightAndBottomConstraints {
 }
 
 - (CGFloat)getRefsPanelHeight {
-    // TODO: this code could be changed to use size classes instead of orientation which is deprecated from iOS 8
+    WMF_TECH_DEBT_WARN(use size classes instead of interfaceOrientation)
+#pragma clang diagnostic push
```

Comment:
@montehurd clang push/pop allows you to enable/disable warnings over a range of code